.circleci/config.yml

```diff
@@ -39,6 +40,7 @@ jobs:
     working_directory: ~/qpp-conversion-tool
     docker:
       - image: circleci/node:7.10.0-browsers
+    resource_class: large
```

Comment:
On second thought, though, we don't really need the `large` resource class for `ui_test` and `ui_deployment` because we're only running into problems with `build`.